test/unit/storage_async_test.rb

```diff
@@ -308,11 +308,16 @@ def test_acl_tls
 
   private
 
+  def create_ca(alg)
```

Comment:
Two things:

1. This method fits better under `TestHelpers::Certificates` where `create_key` and `create_cert` are
2. The parameter `alg` is unused

Reply:
I think so but I don't want to figure out now where we have non-dried methods to create cert files. Also if this has to be correct, the `#create_cert` should sign with the CA. Now CA and certs have nothing to do with each other. And both are marked as CAs... I'm just extracting the logic as a first step, then we can move to helpers once apisonator is working. Anyway, will push a change to at least make `alg` used.